pkg/cmd/evict_pod.go

```diff
@@ -119,3 +136,25 @@ func getNamespace(flags *genericclioptions.ConfigFlags) string {
 	}
 	return namespace
 }
+
+// Evict tries to evict a pod
+func Evict(kubeclient kubernetes.Interface, podname, namespace string) error {
```

Comment:
is there a specific reason to move this function to `main`?

Reply:
felt awkward to have a package with 1 small function in it

Reply:
I understand. Its just that I follow this same pattern with other projects (in this case it is a small fn, but others may have more complex functionality). if its not too much of a bother, I would like to keep it that way. but if you feel very strongly about it, we can move it to main.

Reply:
listPods should be in the same package or it's own ?

Reply:
it could go to same k8s package. we could either rename the file to k8s.go or create a new file. i am ok with either of those. 

thanks again for your time to help improve this project.

Reply:
done